bedrock/firefox/templates/firefox/personal.html

```diff
@@ -37,6 +37,9 @@
     <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
       {{ high_res_img('firefox/android/btn-google-play.png', {'alt': 'Get it on Google Play', 'width': '172', 'l10n': True}) }}
     </a>
+    <small class="download-other">
+      <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+    </small>
```

Comment:
Okay I've increased the font size as suggested in earlier thread above to .75rem and updated the colour as well. Should be better to read now. I agree the links under the download buttons are a little small ><.